conformity/account_settings.go

```diff
@@ -442,13 +515,90 @@ func processRuleValues(vs []interface{}) []*cloudconformity.RuleSettingValues {
 	values := make([]*cloudconformity.RuleSettingValues, 0, len(vs))
 	for _, v := range vs {
 
-		profileValues := &cloudconformity.RuleSettingValues{}
+		ruleSettingValues := &cloudconformity.RuleSettingValues{}
 		item := v.(map[string]interface{})
 
-		profileValues.Enabled = item["enabled"].(bool)
-		profileValues.Label = item["label"].(string)
-		profileValues.Value = item["value"].(string)
-		values = append(values, profileValues)
+		ruleSettingValues.Enabled = item["enabled"].(bool)
+		ruleSettingValues.Label = item["label"].(string)
+		ruleSettingValues.Value = item["value"].(string)
+
+		values = append(values, ruleSettingValues)
+	}
+	return values
+}
+
+func processRuleValuesSettings(vs []interface{}) []*cloudconformity.RuleSettingValues {
+	values := make([]*cloudconformity.RuleSettingValues, 0, len(vs))
+	for _, v := range vs {
+
+		ruleSettingValues := &cloudconformity.RuleSettingValues{}
+		item := v.(map[string]interface{})
+
+		if val, ok := item["enabled"]; ok && val != nil {
+			ruleSettingValues.Enabled = val.(bool)
+		}
+
+		if val, ok := item["customised"]; ok && val != nil {
+			ruleSettingValues.Customised = val.(bool)
+		}
+
+		if val, ok := item["label"].(string); ok && val != "" {
+			ruleSettingValues.Label = val
+		}
+
+		if val, ok := item["value"].(string); ok && val != "" {
+			ruleSettingValues.Value = val
+		}
```

Comment:
🔧 Should use more descriptive variable names here
```suggestion
		if enabled, ok := item["enabled"]; ok && enabled != nil {
			ruleSettingValues.Enabled = enabled.(bool)
		}

		if customised, ok := item["customised"]; ok && customised != nil {
			ruleSettingValues.Customised = customised.(bool)
		}

		if label, ok := item["label"].(string); ok && label != "" {
			ruleSettingValues.Label = label
		}

		if value, ok := item["value"].(string); ok && value != "" {
			ruleSettingValues.Value = value
		}
```